README.md

```diff
@@ -6,18 +6,17 @@ In the face of natural disasters, such as wildfires, Pyrrha (created by Prometeo
 
 
 ## Pyrrha Team
-| Name | Title | Email | Slack |
+| Name | Title | Github | Slack |
 | --- | --- | --- | --- |
-| Upkar Lidder | Technical Lead | Ulidder@us.ibm.com | @Upkarlidder |
-| Daniel Krook | CTO | Krook@us.ibm.com | @Krook |
-| John Segrave | Lead Data Scientist & Technical Product Manager | John.segrave@ie.ibm.com | @Jsegrave |
-| Marco Emilio Rodriguez Serrano | --- | Marco.Rodriguez.Serrano@ibm.com | @Marco.rodriguez.serra |
-| Salome Valero Cumplido | --- | Salome.valero@es.ibm.com | @Salaome.valero |
-| Taeson Black | Tech. Solution Specialist Intern | Tae.Black@ibm.com | @Tae.black|
-| Alejandro Quijada | Tech. Solution Specialist Intern | Alejandro.Quijada@ibm.com | @Alejandro.Quijada |
+| Upkar Lidder | Technical Lead | [Upkar](https://github.com/upkarlidder) | @Upkarlidder |
+| John Segrave | Lead Data Scientist & Technical Product Manager | [John](https://github.com/JSegrave-IBM) | @Jsegrave |
+| Marco Emilio Rodriguez Serrano | --- | [Marco](https://github.com/mrodrise) | @Marco.rodriguez.serra |
+| Salome Valero Cumplido | --- | [Salome](https://github.com/svaleroc) | @Salaome.valero |
+| Taeson Black | Tech. Solution Specialist Intern | [Tae](https://github.com/TaeBlack) | @Tae.black|
+| Alejandro Quijada | Tech. Solution Specialist Intern | [Alejandro](https://github.com/Aquijada52) | @Alejandro.Quijada |
 
 ## IBM Call for Code Team
-| Name | Title | Email | Slack |
+| Name | Title | github | Slack |
```

Comment:
Please capitalize to "GitHub"

Reply:
so you do want seperate team blocks ; one for pyrrha, cfc , and other @krook

Reply:
Yes please